corehq/apps/reports/standard/deployments.py

```diff
@@ -303,8 +306,11 @@ def process_rows(self, users, fmt_for_export=False):
 
         for user in users:
             last_build = last_seen = last_sub = last_sync = last_sync_date = app_name = commcare_version = None
-            last_build_profile_name = None
+            last_build_profile_name = device = device_app_meta = num_unsent_forms = None
             build_version = _("Unknown")
+            devices = user.get('devices', None)
+            if devices:
```

Comment:
you can also just ignore doing this for a web user, devices are only updated for commcare users.

https://github.com/dimagi/commcare-hq/blob/53fa85c2f8bf5f580e86928454509261b40c8445/corehq/apps/users/util.py#L225